### PR TITLE
Improve support for Kubernetes

### PR DESCRIPTION
**Pull request by ananace**
So I found your package and decided to set it up as a minimal jumphost for my local network, but there were two issues that I ran into almost instantly. They can be worked around, but I'd rather see them fixed in the image itself as they're both quite minor things.

So this PR fixes some issues when mounting secrets for host and user keys, as Kubernetes will mount the directory as read-only, and normally ignores any ending newlines for values when you create them.

Basically, if you mount your host keys as `/etc/dropbear` then the `chmod` call will fail, and if you create your `authorized_keys.d` secret with `kubectl create secret generic --from-file` then the concatenation will end up with all keys onto the same line.

### CODE REVIEW

File: scripts/bin/init [hunk 1]
```diff
@@ -65,6 +65,7 @@ for entry in ${DROPBEAR_USERS?}; do
 		printf -- '%s\n' '# AUTOGENERATED, DO NOT EDIT!' > "${authorizedKeys:?}"
 		for f in "${authorizedKeys:?}.d/"*; do
 			[ -f "${f:?}" ] || continue
+			printf -- '\n%s\n' "# ${f:?}" >> "${authorizedKeys:?}"
```
Comment (hectorm) [inline scripts/bin/init]: Instead of appending the file name, I think it would be better to replace `cat` with `awk 1` to ensure that a line break is always printed.

**Comment by hectorm** [comment]
Thanks for your contribution! LGTM.

File: scripts/bin/init (hunk 1 above)
Comment (ananace) [reply scripts/bin/init]: Yeah, I'm definitely not skilled enough with `awk` to know what `awk 1` implies.  
Prepending the filename though is just something I like to do whenever concatenating multiple sources, including some kind of information from where a particular block is originating from is always helpful if there's any need of debugging at a later point.
